Gemfile

```diff
@@ -3,6 +3,11 @@ source 'https://rubygems.org'
 #   spec.add_runtime_dependency '<name>', [<version requirements>]
 gemspec name: 'metasploit-framework'
 
+gem 'bit-struct', git: 'https://github.com/busterb/bit-struct', branch: 'ruby-2.4'
```

Comment:
please remove the 'bit-struct' and 'method_source' gem line additions here, they should no longer be needed